Example join room page (HTML/CSS, related assets), reorganizing repo

rachel-wg opened the pull request:
- HTML/CSS for join room page (passes automatic accessibility checking done with axe), including fonts/font licenses/images/other asset files 
- Reorganizing repo

rachel-wg committed:
Example join room page (HTML/CSS, related assets), reorganizing repo

- HTML/CSS for join room page, including fonts/images/other asset files
- Reorganizing repo

dru26 commented on aframe-audio-components/join-room.html:
```diff
@@ -0,0 +1,55 @@
+     <!-- CSS for this page -->
+    <link rel="stylesheet" type="text/css" href="content/styles/join-room-style.css" >
+    
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
```
I don't think any of the osc or aframe scripts are needed in this html and will be outdated soon anyways with the osc-rework branch

rachel-wg replied:
got it, just removed!

rachel-wg committed:
Deleted extra script links, formatting

- removing unneeded A-Frame and OSC script links from join-room.html
- Updated formatting of join-room files
